packages/ui/src/components/Button.tsx

```diff
@@ -13,21 +13,29 @@ type Props = PropsWithChildren<{
   // We default to type="button", to avoid form-related bugs.
   // In HTML standard, there's also "reset", but it's rarely useful.
   type?: "submit" | "button";
+  // More precise control for the button's layout. Disables padding and flexbox mode.
+  noLayout?: boolean;
 }>;
 
-export const Button: FC<Props> = ({
+// For internal use only, for now (see ButtonWithDropdown).
```

Comment:
Maybe we could export it, btw, could be useful for other purposes than dropdowns.

Reply:
No rush, can export later if needed.